modules/facilities_api/README.rdoc

```diff
@@ -0,0 +1,10 @@
+= FacilitiesApi
+TODO: Short description and motivation.
+
+== Installation
+Ensure the following line is in the root project's Gemfile:
+
+  gem 'facilitiesapi', path: 'modules/facilitiesapi'
```

Comment:
Was this auto generated? Seems like this should be facilities_api (underscore)

Reply:
It's auto-generated. I had to fix this in several other places but the generator is dropping the `_`